File: exercicios/01/e01.c
#include <stdio.h>

int main () {
	int n1;
    int n2;
    int maior;
    int menor;
    int aux;

	printf("digite o primeiro número: ");
	scanf("%d", &n1);

	printf("digite o segundo número: ");
	scanf("%d", &n2);

    if (n1 > n2) {
        maior = n1;
        menor = n2;
    } else {
        menor = n1;
        maior = n2;
    }

	if (maior == menor){
        printf("Você escolheu o número %d 2 vezes", menor);
	} else {
        printf("Você escolheu os números %d e %d.\n", menor, maior);

        aux = menor + 1;
        if (aux + 1 == maior) {
            printf("Entre eles existe o número %d", aux);
        } else if (aux < maior) {

            printf("Entre eles existem os números: %d", aux);

            aux += 1;
            while(aux < maior) {
                printf(", %d", aux);
                aux += 1;
            }

            printf(".");
        }
    }

	return 0;
}

File: exercicios/01/e01.py
# -*- coding: utf-8 -*-

n1 = input("digite o primeiro número: ")
n2 = input("digite o segundo número: ")

if (n1 == n2):
    print("Você escolheu o número " + str(n1) + " duas vezes.")
else: 
    maior = n1 if n1 > n2 else n2
    menor = n1 if n1 < n2 else n2 

    print("Você escolheu os números " + str(menor) + " e " + str(maior) + ".")
        
    if (maior == menor + 2):
        print("Entre eles existe o número " + str(menor + 1))
    elif (maior > menor + 2):
        print("Entre eles existem os números: " + str(menor + 1)),
        menor = menor + 2
        while(maior > menor):
            print(", " + str(menor)),
            menor += 1
        print (".")
    